Propose definition for 'mansplaining' (#150)
tatianamac/selfdefined: 11ty/definitions/mansplaining.md

---

> title: Mansplaining

Reviewer: ```suggestion
title: Mansplain
```

Reviewer: Changing this to make your part of speech correct! 🤗 Alternately, you can keep mansplaining and change it to a noun, your choice!

Eventually we'll get more sophisticated with the dictionary to include the other, related forms of speech for each word!

> slug: mansplaining

Reviewer: ```suggestion
slug: mansplain
```

speech: verb
defined: true
reading:
- text: "Rebecca Solnit's 'Men Explain Things to Me' (2008)"
  href: https://www.guernicamag.com/rebecca-solnit-men-explain-things-to-me/
- text: "Kim Goodwin's 'Am I mansplaining? graphic"
  href: https://ichef.bbci.co.uk/wwfeatures/live/1600_900/images/live/p0/6f/zj/p06fzjxg.jpg
---

> A man explaining a topic to a woman in a condescending, overconfident way, without acknowledging the fact that the woman often knows more about that topic than the man doing the explaining. 

Reviewer: ```suggestion
when someone, often a man, explains a topic to a woman in a condescending, overconfident way, without acknowledging the fact that the person being explained to, often a woman, often knows more about that topic than the explainer. 
```

Reviewer: While I think it's important that it's usually men doing this to women, I do think that we need to show that mansplaining anyone can do if they have relative or systemic power over somebody else.

Reviewer: I would also suggest adding an "Issues" and "Impact" section, as they're somewhat core to the dictionary. Here is a rough first-pass:

```suggestion
## Issues
Mansplaining places men (usually without authority) in a position of default authority, and places women and gender minoritised people with authority in a position of default subordination, which reinforces the power differential in men's favour.

Mansplaining occupies real time in meetings, leading to men gaining more air time and exposure, despite the stereotype that women speak more.[1](https://slate.com/human-interest/2013/02/do-women-talk-more-the-answer-is-no-but-the-belief-persists-despite-the-evidence.html)[2](https://slate.com/human-interest/2013/02/do-women-talk-more-the-answer-is-no-but-the-belief-persists-despite-the-evidence.html)

## Impact
Mansplaining can reinforce toxic masculinity and rape culture, as it fundamentally is about disbelieving women and gender-minoritised individuals.

Mansplaining contributes to the invalidation and disbelief of any women and gender-minoritised individuals and their credentials. The disbelief and intrinsic questioning of women and gender-minoritised individual especially reinforces systemic bias and injustice against them.

Someone who is repeatedly mansplained can be perceived as lacking credentials or knowledge they very well may have, which can lead to missed career opportunities or systematic exclusion from leadership, for example.
```

Examples include but aren't limited to: a man explaining to a woman how to pronounce her own name, explaining menstruation, or explaining the history of a country with which the woman is associated but the man has never been to.  

## Note

A good way to check if you're mansplaining is to ask yourself whether: 

> (1) the person wants the explanation

Reviewer: ```suggestion
(1) the person explicitly asked for the explanation
```

Reviewer: I think this shift of language emphasises consent-based explanations; I would venture to say a lot of mansplainers would incorrectly assume their subjects would want the explanation 😂😭

> (2) you are assuming incompetence, and, 

Reviewer: ```suggestion
(2) you are assuming incompetence because of the person's gender, race, ability, etc, and, 
```

> (3) how bias affects your interpretation of (1) and (2)

Reviewer: ```suggestion
(3) how your systematic and individual bias affects your interpretation of (1) and (2)
```

> 

Reviewer: ```suggestion
### Suggestions
To avoid mansplaining, a good way is to:
- **Pause.** It's always a good idea to think before you talk/type.
- **Ask yourself if your voice is really adding a necessary perspective.** Is this a post written for an audience you're not a part of? Are you an outside observer? It might be best to leave it be and learn from it internally.
- **Check the person's credentials.** Are you, someone with armchair knowledge of viruses, about to explain something to an epidemiologist? It's not okay to mansplain to anyone, but it's especially infuriating if the person has tangible credentials that you do not, and you're trying to explain to them.
- **Ask for consent. Do not expect a "yes."** "Would you like my thoughts on this subject? May I add my perspective here as an amateur epidemiologist who read one uncited paper?"
- **Leave room for and presume knowledge, reflecting on your own knowledge** "You might already know about this, but I found it surprising that..." (The critical part of this example is that it turns it back on you; "__I__ found it surprising that..."
-- 
```
